Sources/ThreatDetectionCenter.swift

```diff
@@ -1,33 +1,98 @@
 import Foundation
 
 public final class ThreatDetectionCenter {
-    
+
+    /// Will check if jailbreak is present
+    ///
+    /// - Returns:
+    ///  `true`, if device is / was jailbroken;
+    ///  `false` otherwise
+    ///
+    /// More about jailbreak: https://wikipedia.org/wiki/Jailbreak_%28iOS%29
+    ///
+    /// ## Notes
+    /// Should also detect jailbreak, even if the device is in a "safe" mode or
+    /// jailbreak mode is not active / was not properly removed
     public static var areRootPrivilegesDetected: Bool {
         JailbreakDetection.threatDetected()
     }
-    
+
+    /// Will check for an injection tool like Frida
+    ///
+    /// - Returns:
+    ///  `true`, if dynamic hooks are loaded at the time;
+    ///  `false` otherwise
+    ///
+    /// More: https://fingerprint.com/blog/exploring-frida-dynamic-instrumentation-tool-kit/
+    ///
+    /// ## Notes
+    /// By the nature of dynamic hooks, this checks should be made on a regular
```

Comment:
Can you please make this change in here and other places where you have `## Notes`?
```suggestion
    /// > By the nature of dynamic hooks, this checks should be made on a regular
```